### PR TITLE
Adding accessibility clients query for platform accessibility clients

### PR DESCRIPTION
**Pull request by yzen**

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/telemetry_derived/accessibility_clients_v1/query.sql [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+      NULL
+    END
+  ) AS TOUCH_KEYBOARD_AND_HANDWRITING_PANEL,
+  COUNT(DISTINCT CASE WHEN instantiator = 'VoiceOver' THEN client_id ELSE NULL END) AS VOICE_OVER,
+  COUNT(DISTINCT CASE WHEN consumer NOT BETWEEN 0 AND 12 THEN client_id ELSE NULL END) AS OTHER
+FROM
+  a11y_consumers_instantiators
+GROUP BY
+  1,
+  2
```
Comment (jklukas) [inline sql/moz-fx-data-shared-prod/telemetry_derived/accessibility_clients_v1/query.sql:141]: Preferred style is to be explicit about the fields used for grouping, 
```suggestion
  submission_date,
  normalized_channel
```

File: dags.yaml [hunk 2]
```diff
@@ -287,3 +287,12 @@ bqetl_marketing_fetch:
     retries: 2
     retry_delay: 30m
 
+bqetl_desktop_platform:
+  schedule_interval: 0 3 * * *
+  default_args:
+    owner: jklukas@mozilla.com
+    start_date: "2020-12-10"
+    email: ["telemetry-alerts@mozilla.com", "jklukas@mozilla.com"]
```
Comment (jklukas) [inline dags.yaml:295]: It looks like you have your email listed in the DAG itself; that should be reflected in this yaml.

File: sql/moz-fx-data-shared-prod/telemetry_derived/accessibility_clients_v1/query.sql [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    CASE
+    WHEN
+      consumer = 0
+      OR instantiator = 'nvda.exe'
+    THEN
+      client_id
+    ELSE
+      NULL
+    END
+  ) AS NVDA,
```
Comment (jklukas) [inline sql/moz-fx-data-shared-prod/telemetry_derived/accessibility_clients_v1/query.sql:43]: Preferred style is for all column names to be lower_snake_case.

**Review by jklukas — COMMENTED** [review]
This looks pretty close to ready to go. My biggest request is to express this on top of `telemetry.main` rather than main_summary. Let me know if it's unclear to you how to accomplish that.

**Review by jklukas — APPROVED** [review]
LGTM. I'll go ahead and create the underlying table and rebase here. As long as tests all pass, we should be good to merge and it will run tonight.

**Comment by jklukas** [comment]
One extra detail here is that we should have a user-facing view defined in `sql/moz-fx-data-shared-prod/telemetry/accessibility_clients/view.sql` that's a simple:
```
SELECT * FROM `moz-fx-data-shared-prod.telemetry_derived.accessibility_clients_v1`
```

File: sql/moz-fx-data-shared-prod/telemetry_derived/accessibility_clients_v1/view.sql [hunk 4]
```diff
@@ -0,0 +1,4 @@
+SELECT
```
Comment (jklukas) [inline sql/moz-fx-data-shared-prod/telemetry_derived/accessibility_clients_v1/view.sql:1]: This view should be in the same folder, but rather in`sql/moz-fx-data-shared-prod/telemetry/accessibility_clients/view.sql` (no _derived and no _v1).

And I forgot to mention that it needs to be have a CREATE OR REPLACE VIEW statement. Please take a look at other view.sql files under `telemetry` for the correct format.

**Comment by jklukas** [comment]
The underlying table is now created and filled with content for 2020-12-09.

File: sql/moz-fx-data-shared-prod/telemetry_derived/accessibility_clients_v1/view.sql (hunk 4 above)
Comment (yzen) [reply sql/moz-fx-data-shared-prod/telemetry_derived/accessibility_clients_v1/view.sql]: Sorry about that, should be fixed now.

File: dags.yaml [hunk 5]
```diff
@@ -287,3 +287,17 @@ bqetl_marketing_fetch:
     retries: 2
     retry_delay: 30m
 
+bqetl_desktop_platform:
+  schedule_interval: 0 3 * * *
+  default_args:
+    owner: jklukas@mozilla.com
+    start_date: "2020-12-10"
```
Comment (yzen) [inline dags.yaml:294]: @jklukas , a quick follow up, I noticed that when I build a query to use the data from `moz-fx-data-shared-prod.telemetry_derived.accessibility_clients_v1` I only get data starting from December 09, 2020. I was hoping to get the data all the way back to March 01, 2017 (all other start dates in this patch). Was I supposed to have this date as the one from 2017 as well?

**Comment by yzen** [comment]
I also wanted to clarify, I was able to use `moz-fx-data-shared-prod.telemetry_derived.accessibility_clients_v1` but not `moz-fx-data-shared-prod.telemetry.accessibility_clients`.. Would both of them suppose to work? I get this error
```
Error running query: b'{\n "error": {\n "code": 404,\n "message": "Not found: Table moz-fx-data-shared-prod:telemetry.accessibility_clients was not found in location US",\n "errors": [\n {\n "message": "Not found: Table moz-fx-data-shared-prod:telemetry.accessibility_clients was not found in location US",\n "domain": "global",\n "reason": "notFound"\n }\n ],\n "status": "NOT_FOUND"\n }\n}\n'
```

**Comment by jklukas** [comment]
View is now published. I am handling the backfill manually.

**Pull request merged**